### DISCUSSION TITLE
No multi exposure option?

### DISCUSSION THREAD

**Discussion by NekkoArms** [sn4k3/UVtools#350] [Q&A] [answered]
im using an ld-002r and im using a sliced file from lychee and the option for the multi-exposure test isn't there I've run a different exposure test in the past so I know its possible

https://i.imgur.com/AL9K5al.png

**Comment by sn4k3** [comment, marked as answer]
Most sliced ctb files on Lychee are CTBv2, you need v3 at least to be able to run that option.
Slice a dummy file with chitubox or use uvtools -> File -> convert to -> CTBv3  or CTBv4

**Reply by NekkoArms** [reply]
LEGENDARY thanks let me give that a go
